GameOfLife/Bipeds.py

```diff
@@ -27,11 +27,35 @@ def filter_gender(self, value):
         fgend_list.sort()
         return fgend_list
 
-    def filter_generation(self, value):
-        """
-        filter_generation must go through all humans in population and make a new list of humans to return if they match the given generation.
-        It has one generation
+    def filter_generation(self, gen):
         """
+             filter_generation must go through all humans in population and make a new list of humans to return if they match the given generation.
+             It has one generation
+             """
+
+        #key, value
+        gen = gen.lower()
+        gen_list = []
+        gen_mapping = {
+                'genx': range(1965, 1980),
+                'genz': range(1997, 2012),
+                'gena': range(2010, 2020),
+                'mill': range(1981, 1996)
+            }
+        for key, r in gen_mapping.items():
+            if key in gen.lower():
+                print(r)
+                for people in self.pop_list:
+                    if people.birthday in r:
+                        gen_list.append(people)
+                        print(people.name)
+                    else:

```

Comment:
This would be rundadent, If the else is there with continue it changes nothing as the if if not trigger would simply still loop. Continue is mostly used when you want to avoid running code that should come later which isn't the case here.